Makefile

```diff
@@ -267,6 +268,12 @@ endif
 	docker build --pull -t $(BUILD_IMAGE):latest-$(ARCH) . --build-arg BIRD_IMAGE=$(BIRD_IMAGE) --build-arg QEMU_IMAGE=$(CALICO_BUILD) --build-arg ver=$(CALICO_GIT_VER) -f ./Dockerfile.$(ARCH)
 	touch $@
 
+# download BIRD source to include in image.
+bird-source: filesystem/included-source/bird-$(BIRD_VER).tar.gz
```

Comment:
Avoid a phony target by saying instead
`BIRD_SOURCE = filesystem/included-source/bird-$(BIRD_VER).tar.gz`
and then using `$(BIRD_SOURCE)` above instead of `bird-source`.